lib/bump/tasks.rb

```diff
@@ -15,7 +15,14 @@
     end
 
     task bump, :tag do |_task, args|
-      run_bump.call(bump, args)
+      raise RakeArgumentsDeprecatedError,
+        "rake arguments are deprecated, use TAG=false to disable tagging" if args.tag
+      options = {
+        tag: ENV['TAG'],
+        commit: ENV['COMMIT'],
+        bundle: ENV['BUNDLE']
+      }
```

Comment:
it would be nicer to do the conversion here instead of going inside of bump.rb and then doing it there since that also catches users that use `.run` directly ... so make parse_cli_options! public and call it here ?